mobile/lib/providers/album/album.provider.dart

```diff
@@ -25,14 +25,22 @@ class AlbumNotifier extends StateNotifier<List<Album>> {
   }
 
   final AlbumService _albumService;
+  final Isar db;
```

Comment:
please remove this instance variable (not needed/accessed anywhere, yet available externally!)
just keep the constructor as it was